### PR TITLE
Rest-java k6 tests in CI

### LINKED ISSUE #12161: Test k6 tests in CI - rest java
### Problem

k6 tests can have functional issues that we don't catch until late in the testing cycle.

### Solution

Add k6 tests in CI for the rest java module.

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by bilyana-gospodinova**
**Description**:
Add rest-java k6 tests execution as part of the workflow in order to verify the end-to-end functionality. This will be helpful to catch some issues earlier in the testing cycle. The goal is not a complete coverage of all rest-java k6 tests, so the failing ones are excluded for now.

Fixes #12161

### CODE REVIEW

File: test/src/test/java/org/hiero/mirror/test/e2e/acceptance/client/TopicClient.java [hunk 1]
```diff
@@ -49,7 +49,16 @@ public TopicClient(
...
-        deleteAll(topicIds, this::deleteTopic);
+        deleteOrLogEntities(topicIds, this::deleteTopic);
+    }
+
+    @Override
+    protected void logEntities() {
+        for (var topicId : topicIds) {
+            log.info("Skipping cleanup of topic at address {}.", topicId);
+            // Log the values so that they can be parsed in CI and passed to the k6 tests as input.
+            System.out.println("DEFAULT_TOPIC=" + topicId.num);
```
Comment (steven-sheehy) [inline test/src/test/java/org/hiero/mirror/test/e2e/acceptance/client/TopicClient.java]: Why do we log both via logger and system.out? Should be one or the other.

File: .github/workflows/acceptance.yaml [hunk 2]
```diff
@@ -186,11 +187,26 @@ jobs:
...
+
+      - name: Run rest-java tests
+        if: ${{ matrix.stream-type != 'BLOCK' }}
+        uses: grafana/run-k6-action@a15e2072ede004e8d46141e33d7f7dad8ad08d9d # v1
+        env:
+          BASE_URL: "http://127.0.0.1:8084"
+          DEFAULT_DURATION: "3s"
+          DEFAULT_GRACEFUL_STOP: "1s"
+          DEFAULT_VUS: 2
+          RESTJAVA_TEST_EXCLUDE: "^(accounts(NftAllowanceOwner|NftAllowanceSpender|OutstandingAirdrop|PendingAirdrop))|rampUp.*$"
```
Comment (steven-sheehy) [inline .github/workflows/acceptance.yaml]: Is there not an acceptance test that adds these?

File: test/src/test/java/org/hiero/mirror/test/e2e/acceptance/client/TopicClient.java (hunk 1 above)
Comment (bilyana-gospodinova) [reply test/src/test/java/org/hiero/mirror/test/e2e/acceptance/client/TopicClient.java]: The logger is now removed.

File: .github/workflows/acceptance.yaml (hunk 2 above)
Comment (bilyana-gospodinova) [reply .github/workflows/acceptance.yaml]: There are, but also there are steps that delete the allowances at the end or claim the airdrops, so after the execution of the acceptance tests they are no longer available. I will see what I can do without adding a bunch of new tests just to do the k6 setup.

File: .github/workflows/acceptance.yaml [hunk 3]
```diff
@@ -182,25 +186,38 @@ jobs:
             echo "${k6}=${value}" >> $GITHUB_ENV
           done
 
+          declare -A k6MapRestJava
+          k6MapRestJava["DEFAULT_TOPIC"]="DEFAULT_TOPIC_WITH_FEE_ID"
+          k6MapRestJava["PRECOMPILE"]="DEFAULT_ACCOUNT_ID_NFTS_ALLOWANCE_SPENDER"
+          k6MapRestJava["DEFAULT_ACCOUNT_ID_BOB"]="DEFAULT_ACCOUNT_ID_AIRDROP_RECEIVER"
+
```
Comment (steven-sheehy) [inline .github/workflows/acceptance.yaml:193]: We shouldn't need a separate rest java map. Can just add to regular k6Map since they all get added to the GITHUB_ENV in the end anyway.

File: .github/workflows/acceptance.yaml [hunk 4]
```diff
@@ -182,25 +186,38 @@ jobs:
...
-        run: |
-          echo "Port forwarding 8080:80"
-          kubectl port-forward "service/${HELM_RELEASE_NAME}-web3" -n "${SOLO_NAMESPACE}" 8080:80 &
+        uses: grafana/run-k6-action@a15e2072ede004e8d46141e33d7f7dad8ad08d9d # v1
+        env:
+          DEFAULT_ACCOUNT_ID_NFTS_ALLOWANCE_OWNER: "0.0.1003"
+          DEFAULT_ACCOUNT_ID_AIRDROP_SENDER: "0.0.1003"
+        with:
+          path: |
+            ./tools/k6/src/rest-java/apis.js
```
Comment (steven-sheehy) [inline .github/workflows/acceptance.yaml:212]: Can't we just run them all at once in a single step by including multiple paths?

File: .github/workflows/acceptance.yaml (hunk 3 above)
Comment (bilyana-gospodinova) [reply .github/workflows/acceptance.yaml:193]: The need for the second map came because there are duplicate keys in web3 and rest-java. Like:
`k6Map["PRECOMPILE"]="HTS_CONTRACT_ADDRESS"` in web3
`k6MapRestJava["PRECOMPILE"]="DEFAULT_ACCOUNT_ID_NFTS_ALLOWANCE_SPENDER"` in rest-java
If we use only 1 map the value will be overriden.
With adding more tests, I suppose these duplicates will get even more so I introduced a new map.

File: .github/workflows/acceptance.yaml (hunk 4 above)
Comment (bilyana-gospodinova) [reply .github/workflows/acceptance.yaml:212]: We can. I added 2 separate steps because if one of them fails we will know directly in which module there is an issue. Otherwise, we need to go through the logs to check.

**Review by xin-hedera — APPROVED** [review]
LGTM
